frontend/ui/widget/inputdialog.lua

```diff
@@ -130,6 +130,8 @@ local InputDialog = InputContainer:new{
     input_type = nil,
     deny_keyboard_hiding = false, -- don't hide keyboard on tap outside
     enter_callback = nil,
+    strike_callback = nil, -- call this on every keystroke
+    InputText = InputText,
```

Comment:
You could name this attribute:
`input_text_class = InputText -- KOTerm provides TermInputText`